core/templates/pages/exploration-editor-page/editor-tab/exploration-editor-tab.component.html

```diff
@@ -4,7 +4,7 @@
      on-finish="$ctrl.onFinishTutorial()"
      on-skip="$ctrl.onSkipTutorial()">
   <div class="exp-editor-main-editor">
-    <div class="oppia-editor-cards-container">
+    <div class="oppia-editor-cards-container protractor-test-fade-in">
```

Comment:
This probably isn't the only element in the app that fades in, so this class name could get confusing once we are trying to wait for other elements to fade in. How about naming this class so as to describe the element instead, for example `protractor-test-editor-cards-container`?

Reply:
Done